cumulus/parachains/runtimes/assets/asset-hub-westend/src/weights/pallet_staking_async.rs

```diff
@@ -0,0 +1,909 @@
+	/// Proof: `Staking::ForceEra` (`max_values`: Some(1), `max_size`: Some(1), added: 496, mode: `MaxEncodedLen`)
+	/// Storage: UNKNOWN KEY `0x876b5159e787527164dca1c92959a6f1` (r:1 w:0)
+	/// Proof: UNKNOWN KEY `0x876b5159e787527164dca1c92959a6f1` (r:1 w:0)
+	/// Storage: `Staking::VoterSnapshotStatus` (r:0 w:1)
+	/// Proof: `Staking::VoterSnapshotStatus` (`max_values`: Some(1), `max_size`: Some(33), added: 528, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ErasValidatorReward` (r:0 w:1)
+	/// Proof: `Staking::ErasValidatorReward` (`max_values`: None, `max_size`: Some(28), added: 2503, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::NextElectionPage` (r:0 w:1)
+	/// Proof: `Staking::NextElectionPage` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ElectableStashes` (r:0 w:1)
+	/// Proof: `Staking::ElectableStashes` (`max_values`: Some(1), `max_size`: Some(32002), added: 32497, mode: `MaxEncodedLen`)
+	fn rc_on_session_report() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `1092`
+		//  Estimated: `39483`
+		// Minimum execution time: 470_999_000 picoseconds.
+		Weight::from_parts(493_273_000, 0)
+			.saturating_add(Weight::from_parts(0, 39483))
+			.saturating_add(T::DbWeight::get().reads(8))
+			.saturating_add(T::DbWeight::get().writes(7))
+	}
+	/// Storage: `Staking::ErasValidatorPrefs` (r:999 w:999)
+	/// Proof: `Staking::ErasValidatorPrefs` (`max_values`: None, `max_size`: Some(57), added: 2532, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ClaimedRewards` (r:999 w:999)
+	/// Proof: `Staking::ClaimedRewards` (`max_values`: None, `max_size`: Some(1462), added: 3937, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ErasStakersPaged` (r:999 w:999)
+	/// Proof: `Staking::ErasStakersPaged` (`max_values`: None, `max_size`: Some(3152), added: 5627, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ErasStakersOverview` (r:999 w:999)
+	/// Proof: `Staking::ErasStakersOverview` (`max_values`: None, `max_size`: Some(92), added: 2567, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ValidatorCount` (r:1 w:0)
+	/// Proof: `Staking::ValidatorCount` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ErasValidatorReward` (r:0 w:1)
+	/// Proof: `Staking::ErasValidatorReward` (`max_values`: None, `max_size`: Some(28), added: 2503, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ErasRewardPoints` (r:0 w:1)
+	/// Proof: `Staking::ErasRewardPoints` (`max_values`: None, `max_size`: Some(36018), added: 38493, mode: `MaxEncodedLen`)
+	/// Storage: `Staking::ErasTotalStake` (r:0 w:1)
+	/// Proof: `Staking::ErasTotalStake` (`max_values`: None, `max_size`: Some(28), added: 2503, mode: `MaxEncodedLen`)
+	/// The range of component `v` is `[1, 1000]`.
+	fn prune_era(v: u32, ) -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `11801 + v * (279 ±0)`
+		//  Estimated: `1633077 + v * (3937 ±0)`
```

Comment:
Weren't you planning to use `pov_mode = measured` for this? 
The proof size is about 4 MB for Polkadot, which seems acceptable for something that runs once per era. But seems `measured` mode would be pretty reliable here right? Since the storage size is more or less fixed and only generated via system tasks (and not via external user triggered code path).

Reply:
Yeah great point, I was on the fence and went with default, and lost track of it.

Measured would indeed be lower, therefore better. And since we won't have any way of clawbacking weight in on_init/XCM stuff, this 4MB will def not be refunded in any way. 

I will rerun the benchmark only for this one in measured, and replace the weight here.